lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
I think I'm gonna skip it for now. Seems like a lot of work (headers are passed, because of how php_curl uses them, as strings that would need to be parsed pretty thoroughly) for very little gain. If someone overrode these headers, they would probably get bad results, but it would not affect the API itself. Or am I missing an edge case here?